src/quo2/components/buttons/button.cljs

```diff
@@ -255,7 +255,8 @@
                    {:background-color
                     (if (= state :pressed)
                       (colors/theme-colors colors/neutral-100 colors/white)
-                      :transparent)}
+                      :transparent)
+                    :width            size}
```

Comment:
could you elaborate ?

Reply:
https://user-images.githubusercontent.com/28704507/213384303-0c92cc21-49ea-4ee3-81d0-747e1bc69f43.mov

add width to button solves the button background issue when the user navigates back from the discover screen

Reply:
just to be clear, this issue is now gone.. as in the video the problem is still there 😓

Reply:
i this its demo of the issue on the video